app/services/submit_to_app_store.rb

```diff
@@ -35,6 +35,7 @@ def submit(submission)
   end
 
   def notify(submission)
-    ClaimSubmissionMailer.notify(submission).deliver_later! if submission.is_a?(Claim)
+    Nsm::SubmissionMailer.notify(submission).deliver_later! if submission.is_a?(Claim)
+    ::PriorAuthority::SubmissionMailer.notify(submission).deliver_later! if submission.is_a?(PriorAuthorityApplication)
```

Comment:
If REDIS_HOST is not defined, so we're _not_ doing this job asynchronously, I don't think we should deliver the emails asynchronously either. i.e. if we come to this from line 10 instead of line 20, we should be doing `deliver!` not `deliver_later!`